pkg/testutils/sqlutils/sql_runner.go

```diff
@@ -37,6 +37,7 @@ func MakeSQLRunner(tb testing.TB, db *gosql.DB) *SQLRunner {
 
 // Exec is a wrapper around gosql.Exec that kills the test on error.
 func (sr *SQLRunner) Exec(query string, args ...interface{}) gosql.Result {
+	sr.Helper()
 	r, err := sr.DB.Exec(query, args...)
 	if err != nil {
 		file, line, _ := caller.Lookup(1)
```

Comment:
Can't you remove this `file, line, _ := caller.Lookup(1)` now?

Reply:
Done.